### PR TITLE
redirectURL nil check added for handleRedirectURL

### PR DESCRIPTION
**Pull request by brianeunilkim**
It throws nil exception when you have multiple account stores with and
without redirectURL. Nil check must be performed to prevent exception.

### CODE REVIEW

File: Sources/OAuth2Client/NXOAuth2Client.m [hunk 1]
```diff
@@ -305,7 +305,7 @@ - (BOOL)openRedirectURL:(NSURL *)URL;
 // Web Server Flow only
 - (void)requestTokenWithAuthGrant:(NSString *)authGrant redirectURL:(NSURL *)redirectURL;
 {
-    NSAssert1(!authConnection, @"authConnection already running with: %@", authConnection);
+    //NSAssert1(!authConnection, @"authConnection already running with: %@", authConnection);
```
Comment (toto) [inline Sources/OAuth2Client/NXOAuth2Client.m:308]: Please don't commit commented out code. Either you want it in or you don't

File: Sources/OAuth2Client/NXOAuth2AccessToken.m [hunk 2]
```diff
@@ -168,7 +168,8 @@ - (BOOL)doesExpire;
 
 - (BOOL)hasExpired;
 {
-    return ([[NSDate date] earlierDate:expiresAt] == expiresAt);
+    // give it a little buffer to expire date
+    return ([[NSDate dateWithTimeIntervalSinceNow:10] earlierDate:expiresAt] == expiresAt);
```
Comment (toto) [inline Sources/OAuth2Client/NXOAuth2AccessToken.m:172]: This change has nothing to do with the nil check the PR is about AFAIKT. Any reason why this is in here?

**Comment by toto** [comment]
In general there is a lot of commented out code parts. Can you clean up your PR please. Thank you!

**Comment by brianeunilkim** [comment]
@toto Oh, sorry. I didn't mean to do a pull request on my master branch. I'll clean up and do separate pull requests.

**Comment by toto** [comment]
Cool. Thanks. Can you do that against the develop baranch

**Pull request closed without merging**